internal/status/http-statuses.csv

```diff
@@ -59,5 +66,10 @@
 506,5,Variant Also Negotiates (Experimental),RFC2295
 507,5,Insufficient Storage,RFC4918
 508,5,Loop Detected,RFC5842
+509,5,Bandwidth Limit Exceeded,Apache Web Server/cPanel
 510,5,Not Extended,RFC2774
 511,5,Network Authentication Required,RFC6585
+529,5,Site is overloaded,Qualys
+530,5,Site is frozen,Pantheon
+598,5,Network read timeout error (Informal convention),Unofficial
+599,5,Network Connect Timeout Error,Unofficial
```

Comment:
```suggestion
598,5,Network Read Timeout Error,Unknown
599,5,Network Connect Timeout Error,Unknown
```

Reply:
Oh, sorry, I thought you wanted me to! I might have overwritten your changes